.tx/config

```diff
@@ -0,0 +1,27 @@
+[main]
+host = https://www.transifex.com
+
+[beavy.server]
+file_filter = translations/<lang>/beavy_server.json
+source_file = var/server-messages/beavy-server.json
+source_lang = en
+type = KEYVALUEJSON
+
+[beavy.web]
+file_filter = translations/<lang>/beavy_web.json
+source_file = var/react-intl-messages/beavy-core.json
+source_lang = en
+type = KEYVALUEJSON
+
+[beavy.hacker-news-app]
```

Comment:
I feel the naming `beavy.apps.hacker-news` and `beavy.modules.comments` would work better. also because we still allow each of them to have a frontend and backend part, right? so `beavy.modules.comments.web` and `beavy.modules.comments.server`?